recipes/boost/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "1.60.0" %}
+
+package:
+  name: boost
+  version: {{ version }}
+
+source:
+  fn:  boost_{{ version.replace(".", "_") }}.tar.bz2
+  url: http://sourceforge.net/projects/boost/files/boost/{{ version }}/boost_{{ version.replace(".", "_") }}.tar.bz2
```

Comment:
Moments like this I start to wonder if the jinja var is a good idea. I am not against it, but it sounds more trouble than it is worth. (I never really like those vars as the URL becomes impossible to copy-and-paste :wink:)